### ISSUE TITLE
Change link to place hold/my account

### ISSUE THREAD

**Issue by seanluyk** [ualbertalib/NEOSDiscovery#221]
**Is your feature request related to a problem? Please describe.**
The current place hold link goes to the barcode login form, which requires users to sign in multiple times to place holds on different items.

**Describe the solution you'd like**
Log in only required once

**Describe alternatives you've considered**
Even better would be to point to this page so users at UofA/MacEwan/Alberta Innovates/Norquest can use single-sign-on with their institutional credentials.

**Additional context**
Submitted via the feedback form:

"It's annoying to have to enter your log in credentials for every book you place on hold.  It would be more helpful to log in once, and be able to search and place holds while logged in, instead of entering credentials with each hold."

**Comment by seanluyk** [comment]
@weiweishi, a page already exists that @jhennig created that allows users at institutions with SSO to log on that way, and others to do so using the barcode login. Last time I checked with Jeremy he was waiting on a test environment

**Comment by seanluyk** [comment]
Have asked Jeremy to work on a ws version of the My Account login: https://catalogue.library.ualberta.ca/myaccount/

**Comment by seanluyk** [comment]
Checked into this with Jeremy and although it's possible, it isn't without some work to make SSO from other institutions work. In the interest of closing this project and not adding more complexity to it, I think this issue should be closed as it isn't an original requirement of creating a catalog interface for NEOS.
